### PR TITLE
add someshit

### PR DESCRIPTION
**Pull request by adityapawar1**
## What's new in this PR
self explanatory
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

PLEASE MERGE ASAP!!

CC: @jjstnlee

### CODE REVIEW

File: api/supabase/queries/query.ts [hunk 1]
```diff
@@ -41,3 +41,22 @@ export async function queryDefaultImages(category: string) {
...
+export async function getStuff(parameter: any) {
+  const { data: defaultImage, error } = await supabase
+    .from('Projects')
+    .select('*')
+    // .eq('category', parameter)
+    .single();
+
+  console.log('defaultImage', defaultImage);
+  console.log('error', error);
+  console.log('parameter', parameter);
```
Comment (celinechoiii) [inline api/supabase/queries/query.ts:55]: get rid of console.log!

File: api/supabase/queries/query.ts [hunk 2]
```diff
@@ -41,3 +41,22 @@ export async function queryDefaultImages(category: string) {
 
   return defaultImage;
 }
+
+
+export async function getStuff(parameter: any) {
```
Comment (celinechoiii) [inline api/supabase/queries/query.ts:46]: vague func name

File: components/Component/index.tsx [hunk 3]
```diff
@@ -141,13 +141,13 @@ export default function ProjectModal({
     return size;
   };*/
 
-  function convertLastUpdatedDateToString() {
+  function convert() {
```
Comment (celinechoiii) [inline components/Component/index.tsx:144]: terrible func names

File: components/Component/index.tsx [hunk 4]
```diff
@@ -141,13 +141,13 @@ export default function ProjectModal({
...
     if (!last_updated_display) return '';
-    const res = new Date(last_updated_display);
-    const year = String(res.getFullYear());
-    const month = res.toLocaleString('default', { month: 'long' });
-    const day = String(res.getDate()).padStart(2, '0');
-    return `${month} ${day}, ${year}`;
+    const x = new Date(last_updated_display);
```
Comment (celinechoiii) [inline components/Component/index.tsx:146]: what do these variable names mean?

File: components/ProjectsListingModal/index.tsx [hunk 5]
```diff
@@ -28,7 +29,11 @@ export default function ProjectsListingModal({
   searchTerm: string | null;
   setSearchTerm: React.Dispatch<React.SetStateAction<string>>;
 }) {
-  const projectItems = projects?.map((project: Project) => {
+  const [value, set_value] = useState(false);
+  console.log('projects', projects);
+  console.log('help me');
+
+  const meep = projects?.map((project: Project) => {
```
Comment (celinechoiii) [inline components/ProjectsListingModal/index.tsx:36]: no animal sounds

File: components/ProjectsListingModal/index.tsx [hunk 6]
```diff
@@ -28,7 +29,11 @@ export default function ProjectsListingModal({
   searchTerm: string | null;
   setSearchTerm: React.Dispatch<React.SetStateAction<string>>;
 }) {
-  const projectItems = projects?.map((project: Project) => {
+  const [value, set_value] = useState(false);
+  console.log('projects', projects);
+  console.log('help me');
```
Comment (celinechoiii) [inline components/ProjectsListingModal/index.tsx:34]: remove

File: components/ProjectsListingModal/index.tsx [hunk 7]
```diff
@@ -28,7 +29,11 @@ export default function ProjectsListingModal({
   searchTerm: string | null;
   setSearchTerm: React.Dispatch<React.SetStateAction<string>>;
 }) {
-  const projectItems = projects?.map((project: Project) => {
+  const [value, set_value] = useState(false);
```
Comment (celinechoiii) [inline components/ProjectsListingModal/index.tsx:32]: use camelCase, dont need this

File: components/ProjectsListingModal/index.tsx [hunk 8]
```diff
@@ -57,9 +62,33 @@ export default function ProjectsListingModal({
...
+              }}
+            >
+              <SubHeading2>ALL PROJECTS</SubHeading2>
+              {value && (
+                <div
+                  style={{
+                    color: 'rgba(46, 58, 89, 0.75)',
+                    fontFamily: '"Coinbase Mono"',
+                    fontSize: '10px',
+                    margin: "-100px",
```
Comment (celinechoiii) [inline components/ProjectsListingModal/index.tsx:80]: nope

**Review by celinechoiii — CHANGES_REQUESTED** [review]
nope
